common/metrics/src/prometheus.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright 2022-2024 Protocol Labs
+// SPDX-License-Identifier: Apache-2.0, MIT
+//! Prometheus metrics
+
+#[macro_export]
+macro_rules! metrics {
```

Comment:
If we export these then they should be supported with docs and examples. Previously it was just a local macro, so the code itself was there to make it clear how it was working, but without that context it's always helpful to explain how it's supposed to be used.